pkgs/applications/misc/paperjam/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib , stdenv, fetchgit, qpdf, libpaper, asciidoc, }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "paperjam";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "git://git.ucw.cz/paperjam.git";
+    # by recording the exact commit, we can embed it into the binary
+    # this can't be done automatically as fetchgit removes .git directory
+    rev = "90173f436b39bfe39491aa356a218ac185618d1c";
+    hash = "sha256-F0bcYXQhtaI48ULhi7o8gLlC0jIuxPIKTq1cABrP2Iw=";
+  };
+
+  buildInputs = [
+    qpdf
+    libpaper
+    asciidoc
+  ];
+
+  makeFlags = [
+    "DESTDIR=$(out)"
+    "PREFIX=/"
```

Comment:
```suggestion
    "PREFIX=$(out)"
```

Reply:
I feel like it better represents the intent, `PREFIX` saying _don't put it under `/usr` but straight under `/`_ and `DESTDIR` saying _do a "chroot" to `$out` during install_, but if you prefer brevity, I'll change it.

Reply:
In Nixpkgs we don't use `DESTDIR`.

`DESTDIR` exists primarily for "targz decompressers" used by other distros. With it, the Makefile compiles the project as if it would be installed under `PREFIX` (including possible hardcoded relationships like the place of libraries and configfiles), but throws the files in a DESTDIR in order that the distro packagers will post-process it (by including metadata, rpm style).

This is called __the poor bois chroot__.

However `nix` installs its things directly in the filesystem. We don't targz things.

Therefore, using `DESTDIR` here is not only superfluous, it is fundamentally wrong.